--- xc7/conda/envs/xc7/lib/python3.7/site-packages/f4pga/utils/vpr_io_place.py	2022-12-02 18:08:17.073970430 -0500
+++ xc7/conda/envs/xc7/lib/python3.7/site-packages/f4pga/utils/vpr_io_place.py	2022-12-02 18:08:31.878078274 -0500
@@ -210,6 +210,8 @@
             )
 
     def output_io_place(self, f):
+        if(len(self.constraints) == 0):
+            return
         max_name_length = max(len(c.name) for c in self.constraints.values())
         print(HEADER_TEMPLATE.format(name='Block Name', nl=max_name_length, s=''), file=f)
 
