diff --git a/xdc-plugin/xdc.cc b/xdc-plugin/xdc.cc
index 9a66029..ee3da55 100644
--- a/xdc-plugin/xdc.cc
+++ b/xdc-plugin/xdc.cc
@@ -228,7 +228,9 @@ struct SetProperty : public Pass {
 
         std::string parameter(args.at(0));
         if (args.size() < 3 || args.at(2).size() == 0) {
-            log_error("set_property %s: Incorrect number of arguments.\n", parameter.c_str());
+	    // warning so I don't need to filter my xdc file
+            log_warning("set_property %s: Incorrect number of arguments.\n", parameter.c_str());
+	    return;
         }
 
         std::string port_name(args.at(2));
